server.js

```diff
@@ -256,6 +263,11 @@ function SocketIOFileUploadServer() {
 								memo: "self-thrown from progress event"
 							});
 						}
+                        else {
```

Comment:
Ah, I only use the the auto-saving feature, so didn't think about that. Fixed in new commit.